content/1.3-migration.md

```diff
@@ -42,8 +42,26 @@ import { EJSON } from 'meteor/ejson';
 meteor npm install --save meteor-node-stubs
 ```
 
-- If you are using app-local packages to control load order and write unit tests for your application, we recommend you switch to using modules. You can read about our recommended structure for applications and modules in the [Application Structure article](structure.html) of the Meteor Guide, and how to test them in the [Testing article](testing.html).
+- If you are using app-local packages to control load order and write unit tests for your application, we recommend you switch to using modules:
+ - Remove code related to the [Package API](http://docs.meteor.com/#/full/packagejs) from the `package.js` files and rename them to `index.js`, 
+ - Move your local packages to the `imports/` directory.
+ - Add the necessary import statements to each of the modules in your packages.
 
+```js
+api.use('meteor');
+// Turns into
+import { Meteor } from 'meteor/meteor';
```

Comment:
I don't think there is any reason to have your `index.js` import these dependencies, right?